app/assets/stylesheets/fonts.css.scss

```diff
@@ -0,0 +1,4 @@
+@font-face {
+       font-family: 'jungle';
```

Comment:
Line should be indented 2 spaces, but was indented 7 spaces<br>Prefer double-quoted strings